x/collection/keeper/msg_server_test.go

```diff
@@ -1263,18 +1386,36 @@ func (s *KeeperTestSuite) TestMsgOperatorBurnFT() {
 			}
 
 			s.Require().NotNil(res)
-
-			if s.deterministic {
-				s.Require().Equal(tc.events, ctx.EventManager().Events())
-			}
+			s.Require().Equal(tc.events, ctx.EventManager().Events())
 		})
 	}
 }
 
 func (s *KeeperTestSuite) TestMsgBurnNFT() {
-	tokenIDs := []string{
-		collection.NewNFTID(s.nftClassID, s.numNFTs*2+1),
+	tokenID := collection.NewNFTID(s.nftClassID, s.numNFTs*2+1)
```

Comment:
Is this for making expectedTokenIDs and expectedAmount?
You may extract this block as a method with a descriptive name then it would be more readable.